eclipse-theia/theia
# Remote: still use settings and configuration from the local user dir

#### What it does

This makes it so that when connected to a remote system or dev-container, configuration from the user dir like tasks, settings, ai-prompts, etc. are still used instead of the configurations located on the remote system/container.

Fixes #14291

#### How to test

Connect to a remote system and see you theme, settings etc are still all set to what they were before connecting.

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
